libs/amount/amount.cpp

```diff
@@ -33,13 +34,36 @@ namespace iroha {
     return result;
   }
 
+  uint256_t getJointUint256(uint64_t first, uint64_t second, uint64_t third,
+                            uint64_t fourth) {
+    uint256_t res(0);
+    res |= first;
+    res <<= 64;
+    res |= second;
+    res <<= 64;
+    res |= third;
+    res <<= 64;
+    res |= fourth;
+    return res;
+  }
+
   Amount::Amount() : value_(0), precision_(0) {}
```

Comment:
Why not setting default value in header file?

Reply:
even better to write in header:
```
uint64_t value_{0};
```